examples/ale_regression_boston.ipynb

```diff
@@ -0,0 +1,790 @@
+{
```

Comment:
<ul><li>Could start a new sentence AFTER "with respect to the mean feature effect". Now I don't really get what you're comparing the magnitude and shape against ? The previous example for linear model? If so, you've already said plots are nonlinear and the reader can hopefully understand this is a different model and draw this conclusion 4 themselves. No need for it to be spelled out. </li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>

Reply:
<p>I like spelling things out, I do kind of want to draw attention to shapes and magnitudes. Rephrased:</p><p> </p><p>"Because the model is no longer linear, the ALE plots are non-linear also and in some cases also non-monotonic. The interpretation of the plots is still the same—the ALE value at a point is the relative feature effect with respect to the mean feature effect, however the non-linear model used shows that the feature effects differ both in shape and magnitude when compared to the linear model."</p>